Dockerfile.template

```diff
@@ -1,44 +1,90 @@
+{{
+	def is_alpine:
+		env.variant | startswith("alpine")
+-}}
+{{ if is_alpine then ( -}}
+FROM ruby:{{ .ruby.version }}-{{ env.variant }}
+{{ ) else ( -}}
 FROM ruby:{{ .ruby.version }}-slim-{{ env.variant }}
+{{ ) end -}}
 
 # explicitly set uid/gid to guarantee that it won't change in the future
 # the values 999:999 are identical to the current user/group id assigned
+{{ if is_alpine then ( -}}
+# alpine already has a gid 999, so we'll use the next id
+RUN addgroup -S -g 1000 redmine && adduser -S -H -G redmine -u 999 redmine
+{{ ) else ( -}}
 RUN groupadd -r -g 999 redmine && useradd -r -g redmine -u 999 redmine
+{{ ) end -}}
 
 RUN set -eux; \
+{{ if is_alpine then ( -}}
+	apk add --no-cache \
+{{ ) else ( -}}
 	apt-get update; \
 	apt-get install -y --no-install-recommends \
-		ca-certificates \
-		curl \
-		wget \
-		\
-		bzr \
-		git \
-		mercurial \
-		openssh-client \
-		subversion \
-		\
-# we need "gsfonts" for generating PNGs of Gantt charts
-# and "ghostscript" for creating PDF thumbnails (in 4.1+)
-		ghostscript \
-		gsfonts \
-		imagemagick \
-# grab tini for signal processing and zombie killing
-		tini \
+{{ ) end -}}
+{{
+	[
+		[
+			# common packages
+			"ca-certificates",
+			"ghostscript", # for creating PDF thumbnails
+			"git",
+			"imagemagick",
+			"mercurial",
+			"openssh-client",
+			"subversion",
+			"tini", # grab tini for signal processing and zombie killing
+			"wget",
```

Comment:
I don't love this (relying on a set of packages with the same name), but the alternative (copying the full list) is worse. :joy: :sob: :heart:

It's even more dramatic in the later set where there's barely any overlap because the two distros use very different naming schemes for dev packages. :see_no_evil: